src/controller/scanner/base_controller.go

```diff
@@ -80,8 +80,9 @@ func (bc *basicController) ListRegistrations(ctx context.Context, query *q.Query
 		return nil, errors.Wrap(err, "api controller: list registrations")
 	}
 	for _, r := range l {
-		if err := bc.appendCap(ctx, r); err != nil {
-			return nil, err
+		if err := bc.RetrieveCap(ctx, r); err != nil {
+			log.Warningf("failed to get capabilities, error: %v", err)
```

Comment:
why this is `Warning` instead of `Error`?

Reply:
User could continue other works when the log level is warn